src/Ordering.Domain/AggregatesModel/OrderAggregate/Order.cs

```diff
@@ -102,7 +102,7 @@ public void SetPaymentId(int id)
 
     public void SetBuyerId(int id)
     {
-        _buyerId = id;
+        BuyerId = id;
```

Comment:
Especially if BuyerId is exposed as a public property, having a public SetBuyerId doesn't seem to make sense - we can just make the setter public.

(this code was written using various DDD-inspired patterns that the team doesn't feel are a good idea)